EKF/estimator_interface.cpp

```diff
@@ -332,6 +388,25 @@ void EstimatorInterface::setRangeData(const rangeSample &range_sample)
 
 		_range_buffer.push(range_sample_new);
 	}
+
+	{
+		// accumulate enough height measurements to be confident in the quality of the data
+		rangeSample sample_delayed;
+
+		if (_range_buffer.peek_first_older_than(_imu_sample_delayed.time_us, &sample_delayed)) {
+			if ((sample_delayed.time_us != 0) && (sample_delayed.time_us > _range_sample_delayed_time_us_last)) {
+				if (_range_hgt_counter == 0) {
+					_range_hgt_lpf.reset(sample_delayed.rng);
+
+				} else {
+					_range_hgt_lpf.update(sample_delayed.rng);
```

Comment:
This is being used anywhere at the moment. The reset function is using the latest measurement which is good for in flight when climbing or descending quickly. We could use the LPF value on startup, but for this type of sensor the best option is a median filter to get rid of spikes.

Reply:
I'll add a median filter.

Reply:
I've updated `resetHeight()` to use the median filtered range data.